src/client/resources/minelabs.client.mixins.json

```diff
@@ -4,7 +4,10 @@
   "compatibilityLevel": "JAVA_17",
   "client": [
     "InGameHudMixin",
```

Comment:
Don't forget to `merge main into <branch>`
The `"InGameHudMixin"` is removed on main and a lot of different changes (just to ensure no breaking conflicts arise)